style.css

```diff
@@ -44,10 +44,61 @@ header {
   text-decoration: none;
 }
 
-nav button {
-  background: transparent;
+/* nav button {
+  position: relative;
   border: none;
   cursor: pointer;
+  z-index: 10;
+} */
+
+.hamburger {
```

Comment:
- [ ] I can see a few differences between your design and Figma design,  please note that it is very important to stick to the Figma design as much as possible in order to have a beautiful website and good user experience 👍. As we can see, your menu option font size and weight, spacing(padding/margin) line height, and position are a bit different, please check the screenshots below ⬇️ to see the differences.  Please make sure that your mobile menu design is similar to the figma template as required for this project 👍

#### Your design
![mobaababa(1)](https://user-images.githubusercontent.com/6615394/223876575-1ae81812-df9b-4708-b2e1-d576970baf96.jpg)

#### figma design
![mobabbahH](https://user-images.githubusercontent.com/6615394/223876670-59f1e6a7-214f-4925-901b-b002c43a9faf.PNG)